Mathlib/MeasureTheory/Measure/OpenPos.lean

```diff
@@ -251,3 +251,44 @@ theorem measure_closedBall_pos (x : X) {r : ℝ≥0∞} (hr : r ≠ 0) : 0 < μ
 #align emetric.measure_closed_ball_pos EMetric.measure_closedBall_pos
 
 end EMetric
+
+section MeasureZero
+/-! ## Meagre sets and measure zero
+In general, neither of meagre and measure zero implies the other.
+- for all $α ∈ (0,1)$, there is a generalised Cantor set $C ⊆ [0,1]$ of measure `α`.
+Cantor sets are nowhere dense. (Taking a countable union of fat Cantor sets whose
+measure approaches 1 yields a meagre set of measure 1.)
+- The set of Liouville numbers is a Lebesgue measure zero subset of ℝ,
+but is dense (in particular, not meagre).
```

Comment:
the comment is still wrong: a dense set can perfectly be meagre. For instance, rational numbers. The fact that Liouville numbers are residual is rather a direct consequence of their definition.